core/scripts/chaincli/command/root.go

```diff
@@ -37,4 +37,5 @@ func init() {
 	RootCmd.AddCommand(feed.RootCmd)
 	RootCmd.AddCommand(BootstrapNodeCmd)
 	RootCmd.AddCommand(RevertReasonCmd)
+	RootCmd.AddCommand(ChainTablesCleanupCmd)
```

Comment:
Hmmm so `core/scripts/chaincli` is not readily available for node ops IIUC. This might belong in the proper node CLI, under `core/cmd`.

Reply:
Didn't know that, will move it